pyproject.toml

```diff
@@ -43,6 +43,8 @@ exclude = ["tests", "docs"]
 [tool.poetry.dependencies]
 python = ">=3.5"
 django = ">=2"
+django-admin-numeric-filter = "^0.1.6"
```

Comment:
We dont want to add these here, as it will force everyone who uses jazzmin to have them installed ..

best to put them under the dev dependencies see(`# 3rd party app support demonstrated in test app`)